sites/all/modules/reol_carousel/js/reol_carousel.js

```diff
@@ -203,8 +203,8 @@
         // Remove placeholders.
         $(swiper.el).find('.ding-carousel-item.placeholder').remove();
         if (data.content) {
-          Drupal.attachBehaviors(data.content);
           swiper.appendSlide(data.content);
+          Drupal.attachBehaviors(data.content);
```

Comment:
Why has this line been moved?

Reply:
You need to add slides to the dom before running drupal attach behaviors, so other has an change to interacte with the elements in the dom. If not the things you run behaviors on are not ready yet...

Reply:
I hade a lot of issues until I found this. The attached functions would be executed but the images would not exists in the dom (spend too much time on this bug :-)) in the lazy load JS code. But when I added an `setTimeout` it would. So found that all attachBehaviours was executed and first then the slides/covers would be added to the dom.